src/acceleration/impl/ResidualPreconditioner.cpp

```diff
@@ -1,9 +1,9 @@
-#include "ResidualPreconditioner.hpp"
+#include "acceleration/impl/ResidualPreconditioner.hpp"
```

Comment:
```suggestion
#include "ResidualPreconditioner.hpp"
```
and a few similar cases. Should not be needed, right?

Reply:
I prefer to keep the path explicit as we add `src/` to our include dirs.
This should prevent conflicts with other headers lying around on the system.